idflakies-legacy/src/main/java/edu/illinois/cs/dt/tools/detection/LegacyDetectorPathManager.java

```diff
@@ -97,6 +103,39 @@ public Path cachePathInstance() {
         }
     }
 
+    @Override
+    public Path startsPathInstance() {
+        String outputPath = Configuration.config().properties().getProperty("dt.incdetector.startspath", "");
+        Logger.getGlobal().log(Level.FINE, "Accessing startspath: " + outputPath);
+        if (outputPath == "") {
+            return modulePath().resolve(".starts");
+        } else {
+            Path outputPathObj = Paths.get(outputPath);
+            try {
+                Files.createDirectories(outputPathObj);
```

Comment:
Since we expect these selection results to be provided as input, we probably should not be trying to create any directories; if they do not exist, should indicate as such (return empty `Path` or `null`?), and the caller should adjust depending on the return.